client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -110,6 +111,11 @@ const useInvestigationTable = (parameters: useInvestigationTableParameters): use
     const axiosInterceptorId = useSelector<StoreStateType, number>(state => state.investigation.axiosInterceptorId);
     const isInInvestigations = useSelector<StoreStateType, boolean>(state => state.isInInvestigation);
 
+    const bc = new BroadcastChannel(BC_TABS_NAME);
```

Comment:
You should define the `BroadcastChannel` in a useEffect so it is initialized only once